docs/examples/model_repository/resnet_torchscript/Readme.md

```diff
@@ -0,0 +1,154 @@
+# Loading Pytorch Models To Nvidia Triton Inference Server
+
+Pytorch models are supported by the Nvidia [Triton Inference Server](https://github.com/triton-inference-server/). To Load the Pytorch model to triton inference server one must convert the model either ONNX format or in the TorchScript format.
```

Comment:
PyTorch models defined in Python are not directly supported by [Triton Inference Server](https://github.com/triton-inference-server/server) For Triton Inference Server to load and run any PyTorch models defined in Python you must convert the model to either [ONNX](https://pytorch.org/tutorials/advanced/super_resolution_with_onnxruntime.html) format or the [TorchScript](https://pytorch.org/docs/stable/jit.html) format.